jenkins.sh

```diff
@@ -14,7 +14,7 @@ rm -rf venv
 virtualenv --never-download venv
 . venv/bin/activate
 # Adding AWS extra to get boto as required by tests
-pip install toil[aws]==3.3.0
+pip install toil[aws]==3.5.0a1.dev256
```

Comment:
Discussed this offline with @cket and @arkal. Versioning check in toil-scripts, toil-lib and toil-rnaseq to consistently use `parse_version`. Versioning check in toil-scripts to narrow the range until BD2KGenomics/toil-lib#46 is merged. Once that PR lands, the versioning check should be removed completely because toil-scripts can rely on toil-lib's setup.py to perform the versioning check.